app/assets/stylesheets/themes/scinote.scss

```diff
@@ -1727,6 +1727,17 @@ th.custom-field .modal-tooltiptext {
   max-height: 200px;
 }
 
+a.disabled {
+  opacity: 0.5;
+  pointer-events: none;
+  cursor: default;
+}
+
+a.disabled-with-click-events {
+  opacity: 0.5;
+  cursor: wait;
```

Comment:
Maybe I'd change this to `not-allowed`, I think it would be more consistent with the rest of cursor in sciNote.

Reply:
I would insist - sorry 😃. My reasoning is that I don't want to introduce another cursor, bootstrap uses `not-allowed` for various disabled things by default (e.g. disabled forms, buttons, etc.), so I just feel this would be more consistent with the rest of the application.